test/python/tensorflow/tests_linux_ie_cpu.txt

```diff
@@ -28,38 +28,30 @@ tests_common.txt
 
 ###################################################
 [SKIP]
-# Specify tests/patterns/regex that should be excluded/skipped
+
```

Comment:
Are there tests enabled or just moved as I see additions as well?

Reply:
There aren't supposed to be any additions. The tests are just moved as this change enabled many of them.